assets/styles/_typography.scss

```diff
@@ -8,6 +8,15 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+ol li, ul li {
+  list-style-position: inside;
```

Comment:
I thought we were binning this?

Reply:
Binning it breaks all the li's everywhere (dashboard, breadcrumbs, etc).